source/keyboardHandler.py

```diff
@@ -369,22 +369,17 @@ def _get_mainKeyName(self):
 		return winUser.getKeyNameText(self.scanCode, self.isExtended)
 
 	def _get_modifierNames(self):
-		modTexts = set()
+		modTexts = []
 		for modVk, modExt in self.generalizedModifiers:
 			if isNVDAModifierKey(modVk, modExt):
-				modTexts.add("NVDA")
+				modTexts.append("NVDA")

```

Comment:
Does this need to be uppercase for some reason? It will get converted to lowercase right? Or is it for the log output?